.github/actions/markPullRequestsAsDeployed/index.js

```diff
@@ -54,18 +54,29 @@ message += `\n\n platform | result \n ---|--- \n🤖 android 🤖|${androidResul
 message += `\n🍎 iOS 🍎|${iOSResult} \n🕸 web 🕸|${webResult}`;
 
 /**
- * Create comment on each pull request
+ * Comment Single PR
+ *
+ * @param {Object} pr
+ * @returns {Promise<void>}
  */
-prList.forEach((pr) => {
-    githubUtils.createComment(github.context.repo.repo, pr, message, octokit)
+function commentPR(pr) {
+    return githubUtils.createComment(github.context.repo.repo, pr, message, octokit)
         .then(() => {
             console.log(`Comment created on #${pr} successfully 🎉`);
+
+            // Sleep for 1 sec before making another request
+            return new Promise(res => setTimeout(res, 1000));
```

Comment:
Can we somehow check the GH API's response headers somehow? They should include a set of various headers about rate-limiting, like [`X-RateLimit-Remaining`](https://docs.github.com/en/rest/overview/resources-in-the-rest-api), which would tell us whether we need this sleep timeout.
Might be overkill for now though, I'll defer to @roryabraham

Reply:
I like that idea @francoisl

Reply:
Let me see then it octokit exposes it for this method.

Reply:
Just found a nice plugin to throttle request when RateLimit is hit. https://github.com/octokit/plugin-throttling.js. We can then remove 1-sec wait limit.

Reply:
@parasharrajat, just make sure it works with the [`@actions/github`](https://github.com/actions/toolkit/tree/main/packages/github) package.